django/chat/templates/chat/components/chat_options_accordion.html

```diff
@@ -8,7 +8,12 @@
 </div>
 <script>
   (function (){
+    {% if prompt %}document.querySelector('#chat-prompt').value = '{{ prompt }}'{% endif %}
```

Comment:
This will run when a preset is loaded or reset - and also when the chat initially loads..